.github/workflows/tests.yaml

```diff
@@ -103,3 +103,12 @@ jobs:
           test -f *.charm
           sg lxd -c "charmcraft -v clean --project-dir ../charm-smoke-test"
           popd
+
+          mkdir -p destructive-mode-tests
+          pushd destructive-mode-tests
+          source /etc/os-release
+          charmcraft -v init --author testuser
+          sed -i "s|20.04|$VERSION_ID|g" charmcraft.yaml
+          charmcraft -v pack --destructive-mode
+          test -f "destructive-mode-tests_ubuntu-$VERSION_ID-amd64.charm"
```

Comment:
At the moment the only effect of clean in this scenario would match the previous test, that is clean the LXD instances.  I originally had it but removed it because it would require the sg wrapper which looked out of place here.